docs/getting-started-apm-server.asciidoc

```diff
@@ -0,0 +1,285 @@
+
+If you'd rather install the stack yourself, first see the https://www.elastic.co/support/matrix[Elastic Support Matrix] for information about supported operating systems and product compatibility.
+
+You'll need:
+
+* *{es}* for storing and indexing data.
+* *{kib}* for visualizing with the APM UI.
+
+We recommend you use the same version of Elasticsearch, Kibana, and APM Server.
+
+image::images/apm-architecture-diy.png[Install Elastic APM yourself]
+
+See {stack-gs}/get-started-elastic-stack.html[Getting started with the {stack}]
+for more information about installing these products.
+After installing the {stack}, read the following topics to learn how to install,
+configure, and start APM Server:
+
+* <<installing>>
+* <<apm-server-configuration>>
+* <<apm-server-starting>>
+* <<next-steps>>
+
+// *******************************************************
+// STEP 1
+// *******************************************************
+
+[[installing]]
+=== Step 1: Install APM Server
+
+NOTE: *Before you begin*: If you haven't installed the {stack}, do that now.
+See {stack-gs}/get-started-elastic-stack.html[Getting started with the {stack}].
+
+To download and install {beatname_uc}, use the commands below that work with your system.
+If you use Apt or Yum, you can <<setup-repositories,install {beatname_uc} from our repositories>>
+to update to the newest version more easily.
+
+ifeval::["{release-state}"!="unreleased"]
+See our https://www.elastic.co/downloads/apm[download page]
+for other installation options, such as 32-bit images.
+endif::[]
+
+[[deb]]
+*deb:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of APM Server has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+["source","sh",subs="attributes"]
+----------------------------------------------------------------------
+curl -L -O {downloads}/apm-server-{version}-amd64.deb
+sudo dpkg -i apm-server-{version}-amd64.deb
+----------------------------------------------------------------------
+
+endif::[]
+
+[[rpm]]
+*rpm:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of APM Server has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+["source","sh",subs="attributes"]
+----------------------------------------------------------------------
+curl -L -O {downloads}/apm-server-{version}-x86_64.rpm
+sudo rpm -vi apm-server-{version}-x86_64.rpm
+----------------------------------------------------------------------
+
+endif::[]
+
+[[mac]]
+*mac:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of APM Server has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+["source","sh",subs="attributes"]
+------------------------------------------------
+curl -L -O {downloads}/apm-server-{version}-darwin-x86_64.tar.gz
+tar xzvf apm-server-{version}-darwin-x86_64.tar.gz
+------------------------------------------------
+
+endif::[]
+
+[[linux]]
```

Comment:
I'd reconsider the order of the sections to `deb`, `rpm`, `linux`, `mac`, `windows`, and `Running on Docker`:

Maybe is a good idea also to rename `linux` to `other linux`, and change casing to `Linux`, `Mac`, `Windows`, and maybe `RPM` and `DEB` too.

If you change anything here, make sure it is consistent elsewhere... I am sort of sensing that you have your good reasons to use lower case there, but eg. in http://apm-server_2170.docs-preview.app.elstc.co/guide/en/apm/server/master/running-with-systemd.html is `RPM` and `DEB` :)

Speaking of which: http://apm-server_2170.docs-preview.app.elstc.co/guide/en/apm/server/master/running-with-systemd.html shouldn't be embedded in or linked to the running section?

Reply:
These are good points. I've made some updates.

From what I can tell, the _correct_ capitalization is `deb` and `RPM`. The only reason I had everything capitalized the other way was because that's how Beats does it... in other words, not a very good reason. I'll update these on our end, then chat with DeDe about updating them in all of the shared Beats docs as well.